### ISSUE TITLE
[Feature request] Universal macOS build

### ISSUE THREAD

**Issue by LenaWil** [RPCS3/rpcs3#16391]
### Quick summary

When the macOS arm64 build is more stable (e.g. doesn’t boot GT6 for me currently, might be my config though) maybe it’s a good idea to provide universal builds too?

**Comment by digant73** [comment]
@main developers is that FR still valid or maybe it can be closed?

**Comment by schm1dtmac** [comment]
> [@main](https://github.com/main) developers is that FR still valid or maybe it can be closed?

I'd say to close it I'm afraid, Intel application support is on its last legs now as of Mac OS X Golden Gate and the next release of the OS will remove Rosetta support barring some exceptions (and said future release's SDK will block building for Intel targets I predict, so eventually we won't be able to build for Intel without forcing it down to an older SDK), given all of that I don't wish to waste the time necessary to either stripe together a hacky Universal build under the current brew-based CI flow, or try and migrate to MacPorts and make Universal builds that way (which I also found to be a painful mess due to dependencies having very spotty coverage for Universal prebuilts, ugh).

**Comment by kd-11** [comment]
Agreed. This is no longer relevant as apple appears to be abandoning intel support.
We'll also be switching to the KK driver in future which doesn't run on the intel devices, so keeping the builds separate is the right call.
